src/main/java/uk/gov/hmcts/reform/et/syaapi/service/NotificationService.java

```diff
@@ -213,6 +214,7 @@ public void sendAcknowledgementEmailToRespondents(CaseData caseData,
             emailToRespondentTemplate = notificationsProperties.getRespondentTseEmailTypeATemplateId();
         }
         respondentParameters.put("linkToDocument", Objects.requireNonNullElse(documentJson, ""));
+        respondentParameters.put("exuiCaseDetailsLink",notificationsProperties.getExuiCaseDetailsLink() + caseId);
```

Comment:
Could we use String.format here? 

Other suggestion, could we write a method in notificationsPropties to generate a link, something like;
getExUiCaseLink(int caseId)

Reply:
If we've created `notificationsProperties.getExuiCaseDetailsLink() ` (ie, not lombok generated) then you could make that method take a parameter for the caseId

Reply:
It is a lombok class, so I went with the string.format option